### ISSUE TITLE
New release (django-fiber 1.5.0)

### ISSUE THREAD

**Issue by richardbarran** [django-fiber/django-fiber#253]
It's probably a good idea to prepare a new release:
- official support for Django 1.11.
- ~~[Move to new Transifex repo](https://github.com/django-fiber/django-fiber/issues/251) (I'll handle it)~~.
- ~~[Updated README](https://github.com/django-fiber/django-fiber/issues/250)~~.
- Fix deprecation warnings.
- Document the release process/guidelines for collaborators in the readthedocs pages?
- And actually release!

**Comment by richardbarran** [comment]
I've started [a page in the wiki for project maintainers](https://github.com/django-fiber/django-fiber/wiki/Project-maintainers'-guide). 
As Django-fiber is going to be maintained by a group of people, I thought that it might be helpful to jot down the release process, so that we're not tied down by having only one person knowing how to make a full release.

**Comment by richardbarran** [comment]
A message to the other project maintainers: I'm looking at releasing a new version of Django-fiber. I've used on other projects [Zest Releaser](https://github.com/zestsoftware/zest.releaser) which is a small set of tools that automates much of the gruntwork of preparing a release: updating the version number, the changelog, uploading to Pypi, tag in the VCS, and more.

Thoughts/comments/suggestions?

**Comment by richardbarran** [comment]
By the way - I've created a new branch 'django20' and started work on making django-fiber compatible with Django 2.0.

**Comment by dbunskoek** [comment]
Looks like django-mptt and Django REST Framework are already pretty close to Django 2.0 support:
https://github.com/django-mptt/django-mptt/issues/613
http://www.django-rest-framework.org/topics/3.7-announcement/#django-20-support

Apart from these dependencies, what else needs to be done?

**Comment by richardbarran** [comment]
The `dev` branch now runs the unit tests on Django 2.0 with 100% success. I will try it out on one of my sites for a few days, then prepare an official release.
If anyone else wants to try it out - please do! :-)

**Comment by richardbarran** [comment]
I have just released Django-fiber 1.5. This includes full support for Django 2.0, some new i18n strings downloaded from Transifex, and dropping support for Django 1.8.

I've also plugged in Zest.releaser in order to automate the release process, and [added a page to the documentation with some notes on the release process](https://django-fiber.readthedocs.io/en/master/development.html).

Now I can finally close this ticket :-)
